etl/steps/data/garden/aviation_safety_network/2022-10-14/aviation_statistics.meta.yml

```diff
@@ -0,0 +1,73 @@
+dataset:
+  namespace: aviation_safety_network
+  version: 2022-10-14
+  title: Fatal aviation accidents and fatalities
+  short_name: aviation_statistics
+  publication_year: 2022
+  publication_date: 2022-10-14
+  description: |
+    Data on fatal accidents, hijacking incidents and numbers of fatalities are based on airliners of 14+ passengers, and do not include corporate jet and military transport accidents.
+    
+    Air traffic figures are based on the number of airliner flight departures (not the number of passengers) per year.
+  sources:
+    -
+      name: Aviation Safety Network
+      published_by: Aviation Safety Network
+      publisher_source: |
+        Information contained in the Aviation Safety Network database is based on information from official sources (authorities, safety boards).
+      publication_year: 2022
+      date_accessed: 2022-10-14
+      url: "https://aviation-safety.net/statistics/"
+    -
+      name: World Bank's World Development Indicators
+      published_by: World Bank's World Development Indicators
+      publication_year: 2022
+      date_accessed: 2022-05-26
+
+tables:
+  aviation_statistics:
+    variables:
+      accidents_per_million_flights:
+        title: Fatal accidents per million commercial flights
+        short_unit: 
+        unit: accidents
```

Comment:
if the variable is a rate I tend to include this in the unit - so I would have 'accidents per million flights'. I think this is more informative as a hover on the map.

Reply:
Same for other rate units below.

Reply:
Thanks. [In this case](https://ourworldindata.org/grapher/air-passengers-per-fatality), there is only global data, for a line chart, and it looks a bit busy with something like "accidents per million flights". I'll see what's best.